bucoffea/monojet/monojetProcessor.py

```diff
@@ -150,7 +153,10 @@ def process(self, df):
         # Already pre-filtered!
         # All leptons are at least loose
         # Check out setup_candidates for filtering details
-        met_pt, met_phi, ak4, bjets, ak8, muons, electrons, taus, photons = setup_candidates(df, cfg)
+        met_pt_dict, met_phi_dict, ak4, bjets, ak8, muons, electrons, taus, photons = setup_candidates(df, cfg)
```

Comment:
Shouldnt there be a VMap returned here?

Reply:
You're right, this is some code left out from older versions of the change. I didn't update the monojetProcessor yet, so it doesn't work properly. For now, I also removed it from the Github Actions testing, but I will try to update the monojetProcessor as well.

Reply:
Ok, let's finalize vbf first and then, when everything is set, we update monojet. Avoid duplicate work.